monkeykitui/src/main/kotlin/com/criptext/monkeykitui/recycler/MonkeyAdapter.kt

```diff
@@ -756,14 +758,6 @@ open class MonkeyAdapter(val mContext: Context, val conversationId: String) : Re
      * Looks for a monkey item with a specified Id, starting by the most recent ones.
      * @return the message with the requested Id. returns null if the message does not exist
      */
-
-
-    fun findMonkeyItemByIdReverse(id: String) : MonkeyItem? = messagesList.get(getItemPositionById(id))
-
-    /**
-     * Looks for a monkey item with a specified Id, starting by the most recent ones.
-     * @return the message with the requested Id. returns null if the message does not exist
-     */
     fun findMonkeyItemById(id: String) = messagesList.findLast { it.getMessageId() == id }
 
```

Comment:
This implementation should be used in `MonkeyItem`'s `findLastPositionById`. It's shorter.
Then, `MonkeyAdapter`'s  `getLastItemPositionById` should call  `MonkeyItem`'s `findLastPositionById`. 
We don't need two implementations that achieve the same result.